### PR TITLE
Single image Eiger data

### PR DESCRIPTION
**Pull request by benjaminhwilliams**
Add a single-image Dectris-flavour Eiger data set, primarily for testing the appropriate DXTBX format class.

A Diamond-Light-Source-flavour Eiger data set is incoming.

### CODE REVIEW

**Comment by dagewa** [comment]
Hi @benjaminhwilliams, thanks for doing the dataset shaving. I note however that the result does not appear to import directly with `dials.import`. Also, it picks up the `FormatHDF5` reader, but really we want to exercise `FormatHDF5EigerNearlyNexus`.

**Comment by benjaminhwilliams** [comment]
> Hi @benjaminhwilliams, thanks for doing the dataset shaving. I note however that the result does not appear to import directly with `dials.import`. Also, it picks up the `FormatHDF5` reader, but really we want to exercise `FormatHDF5EigerNearlyNexus`.

Yep, there's a PR incoming in DXTBX too.  As well as fixing the behaviour of `dials.import`, that will include a test, for which these are the test data.

**Comment by benjaminhwilliams** [comment]
OK, Sorry @dagewa, you had a point.  I had neglected to copy the attributes of the HDF5 groups and datasets from the original data to the semi-synthetic data (in fact, I'd forgotten HDF5 attributes even existed and knew nothing until now about how they're used in NeXus).  That's important because `FormatHDF5` queries the `NX_class` attribute of the top-level `entry` group of the master `.h5` file to determine which format class to use.

That last commit fixes this.  If you check out the branch `better_nexus` in DXTBX and try `dials.import Dectris_Eiger_master.h5`, things should work smoothly.

File: image_examples/README.md [hunk 1]
```diff
@@ -28,3 +28,15 @@ TIMEPIX_SU_512-stdgoni_0001.img
 TIMEPIX_SU_516-stdgoni_0001.img
 - Courtesy of Viktor Bengtsson (2019)
 - For testing FormatSMVTimePix_SU_516x516
+
+Dectris_Nexus_master.h5\
+Dectris_Nexus_data_000001.h5
```
Comment (Anthchirp) [inline image_examples/README.md]:
```suggestion
Dectris_Eiger_master.h5\
Dectris_Eiger_data_000001.h5
```

Comment (Anthchirp) [reply image_examples/README.md]: do we need capital letters at all?

Comment (benjaminhwilliams) [reply image_examples/README.md]: Oops, good spot, thanks.  Thought I'd done that.  But yeah all-lowercase probably better.

**Comment by benjaminhwilliams** [comment]
> Thanks, I can confirm this data set behaves as expected in my hands. Do you have the script you used to cut back to a single image and other file size minimisation operations somewhere public?

Sorry @dagewa, I missed this message.  I'll tidy up the script (it's still a little rough around the edges) and pop it somewhere public shortly.

**Pull request merged**